linkerd/app/src/lib.rs

```diff
@@ -343,8 +343,15 @@ impl App {
                         // Spawn the DNS resolver background task.
                         tokio::spawn(dns.instrument(info_span!("dns")));
 
-                        if let tap::Tap::Enabled { daemon, serve, .. } = tap {
-                            tokio::spawn(daemon.instrument(info_span!("tap")));
+                        if let tap::Tap::Enabled {
+                            registry, serve, ..
+                        } = tap
+                        {
+                            tokio::spawn(
+                                registry
+                                    .clean(tokio::time::interval(Duration::from_millis(100)))
```

Comment:
100ms is a pretty tight poll for something this that should be super rare, and the cost of a stale instance should be pretty light.

```suggestion
                                    .clean(tokio::time::interval(Duration::from_seconds(60)))
```